kytos/core/buffers.py

```diff
@@ -83,18 +92,25 @@ class KytosBuffers(object):
     def __init__(self):
         """Constructor of KytosBuffer build four KytosEventBuffers.
 
-        RawEvent:   KytosEventBuffer with events received from network.
-        MsgInEvent: KytosEventBuffer with events to be received.
-        MsgOutEvent: KytosEventBuffer with events to be sent.
-        AppEvent: KytosEventBuffer with events sent to NApps.
+        :attr:`raw`: :class:`~kytos.core.buffers.KytosEventBuffer` with events
+        received from network.
+
+        :attr:`msg_in`: :class:`~kytos.core.buffers.KytosEventBuffer` with
+        events to be received.
+
+        :attr:`msg_out`: :class:`~kytos.core.buffers.KytosEventBuffer` with
+        events to be sent.
+
+        :attr:`app`: :class:`~kytos.core.buffers.KytosEventBuffer` with events
+        sent to NApps.
         """
         self.raw = KytosEventBuffer('raw_event')
         self.msg_in = KytosEventBuffer('msg_in_event')
         self.msg_out = KytosEventBuffer('msg_out_event')
         self.app = KytosEventBuffer('app_event')
 
     def send_stop_signal(self):
-        """Method usd to send a kytos/core.shutdown event to all buffers."""
+        """Method to send a ``kytos/core.shutdown`` event to all buffers."""
```

Comment:
Since we are rewriting, let's go with:
`Send a ``kytos/core.shutdown`` event to each buffer.`